### DISCUSSION TITLE
主页Unity版本问题

### DISCUSSION THREAD

**Discussion by qq634488405** [jynew/jynew#1267] [General]
主页显示的是如下版本
![image](https://user-images.githubusercontent.com/3225938/196372941-c86ce30a-365a-4a1b-9659-2035a0eb5102.png)
2020.3.32f1c1
实际项目使用的是2020.3.32f1，国际版的Unity，我说怎么下载了2020.3.32f1c1打开还提示要转换版本。。

**Comment by 0kk470** [comment]
其实没什么区别，你可以直接更改ProjectVersion.txt
